packages/spacecat-shared-data-access/src/index.d.ts

```diff
@@ -660,6 +730,11 @@ export interface DataAccess {
   createKeyEvent: (keyEventData: object) => Promise<KeyEvent>;
   getKeyEventsForSite: (siteId: string) => Promise<KeyEvent[]>
   removeKeyEvent: (keyEventId: string) => Promise<void>;
+
+  // experiment functions
+  getExperiments: (siteId: string) => Promise<Experiment[]>;
```

Comment:
signature of this function seem to contain an optional `experimentId` param as per the implementation

```
getExperiments: (siteId: string, experimentId?: string)
```